testing/mongo-c-driver/APKBUILD

```diff
@@ -0,0 +1,58 @@
+# Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
+pkgname=mongo-c-driver
```

Comment:
```suggestion
pkgname="mongo-c-driver"
```

Reply:
Fixed.
I will mark this conversation as resolved.